universal-application-tool-0.0.1/app/export/CsvExporter.java

```diff
@@ -1,9 +1,44 @@
 package export;
 
+import java.io.IOException;
 import java.io.Writer;
+import java.util.List;
+import java.util.Optional;
 import models.Applicant;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
 
 public class CsvExporter implements Exporter {
+  private boolean wroteHeaders;
+  private List<String> headers;
+  private List<String> columns;
+
+  public CsvExporter(List<String> headers, List<String> columns) {
```

Comment:
suggestion (non-blocking): having a single list of `Column` objects with `getHeader()` and `getColumnKey()` would prevent bugs where the two lists are somehow mis-aligned. It'd also leave open the extension of nested columns where a column could know how to serialize its repeated contents if need be.

Reply:
Up to you :-), feel free to resolve either way

Reply:
Did it!